docs/quality-assurance/principles-of-safety-assurance.md

```diff
@@ -0,0 +1,23 @@
+# What is safety ?
+
+So the first question is what is safety in relation to UKHO products:
+
+* An error in a product that could potentially impact the safety of the mariner
+* A delay to the mariner that could potentially impact the safety of the mariner
+
+# Some general principles
+
+* Reliance on manual processes is minimised
+* Best practices in software and test engineering are applied
+* Sound human engineering principles are applied to the design of the software use interface to minmise the probability of human error
+* Software with safety critical functionality must have clear evidance that all safety requirements have been met or relevant 
```

Comment:
I think our stuff is 'safety related' rather than 'safety critical' according to the definition I've seen